### PR TITLE
network: Remove shade hack

### PR DESCRIPTION
**Pull request by almusil**
openstacksdk supports everything that was done through
the shade hack, it is not needed anymore. Remove the hack
completely.

Signed-off-by: Ales Musil <amusil@redhat.com>

### CODE REVIEW

File: network-suite-master/test-scenarios/ovs/test_ovn_physnet.py [hunk 1]
```diff
@@ -56,6 +56,11 @@ def ovn_physnet_small_mtu(
         cluster_network = clusterlib.ClusterNetwork(ovs_cluster)
         cluster_network.assign(network)
         provider_network = _get_network_by_name(default_ovn_provider_client, OVN_PHYSNET_NAME)
+        # default_ovn_provider_client.update_network(
```
Comment (erav) [inline network-suite-master/test-scenarios/ovs/test_ovn_physnet.py:59]: leftover?

Comment (almusil) [reply network-suite-master/test-scenarios/ovs/test_ovn_physnet.py:59]: I was testing if the first part of patch works, we still need https://review.opendev.org/c/openstack/openstacksdk/+/761933 to be merged first

**Review by erav — APPROVED** [review]

**Comment by almusil** [comment]
Also passed full network suite

**Review by erav — APPROVED** [review]

**Pull request merged**